### DISCUSSION TITLE
Pytorch geometric temporal mamba installation

### DISCUSSION THREAD

**Discussion by chara995** [state-spaces/mamba#616] [Q&A]
Hello,

I want to install Pytorch geometric temporal on an environment using the mamba commands. However I only found the pip version to install pytorch geometric temporal. Is there a conda/mamba possibility?

If not,  I am worried about mixing mamba and pip commands. Here is how the installation would look:

module load Mambaforge/23.3.1-1hpc1
mamba create -n GPT_env
mamba activate GPT_env
mamba install pytorch==2.2.2 torchvision=0.17.2 torchaudio==2.2.2 pytorch-cuda=12.1 -c pytorch nvidia
mamba install pyg -c pyg
**pip install torch-geometric-temporal?**
mamba install spyder
…

**Comment by Rainbow96r** [comment]
It's recommended to install all packages available via Mamba first, and only then use pip for the remaining installations, while adhering to this sequence whenever possible.
